molecule/plain-rhel/molecule.yml

```diff
@@ -108,6 +108,9 @@ provisioner:
           - jcustenborder/kafka-connect-spooldir:2.0.43
 
         connect_s3_plugin_version: 10.3.0
+        kafka_connect_plugins:
+          - "/tmp/local_plugins/confluentinc-kafka-connect-gcp-functions-1.1.9.zip"
```

Comment:
this version would also change with CP version, right? won't hard coding fail in that case?

Reply:
this is actually latest connect plugin's version (picked up from here https://www.confluent.io/hub/confluentinc/kafka-connect-gcp-functions)  , I don't think they are dependent or should change with CP version.